packages/composite-checkout/src/components/billing-fields.js

```diff
@@ -88,6 +88,7 @@ function isElligibleForVat() {
 }
 
 function DomainFieldsCheckbox( { toggleVisibility, isDomainContactVisible } ) {
+	const __ = useLocalize();
```

Comment:
I'm not a huge fan of the `__` name — it's not very obvious right away. Could we use `translate` instead?

Reply:
I'm not sure. I don't love the name either, but the way that our translations work is that the files are scanned for the specific function name. If we were using `@automattic/i18n-calypso` we would need to name everything `translate()` but since the function in `@wordpress/i18n` is `__()`, it feels appropriate to use that package's name.

That said, this change doesn't actually add any translation yet (unless it somehow carries over from the monorepo, which it might) and the name might even be configurable, so perhaps it doesn't matter what we use for a name right now.

If that's the case, I might just restore using `localize()` for now and we can rename things later when we have a system in place to scan the files.

Reply:
> since the function in @wordpress/i18n is __(), it feels appropriate to use that package's name.

that does make sense. I'll leave it up to you. We can stick with `__` if that's how other packages use it. I'm also happier with localize :D

Reply:
I removed that change for now. It can be a separate PR.